app/views/collaborators/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Add Collaborator</h1>
+
+<div class="row">
+  <div class="col-md-5">
+    <p>Please enter the email address of the user you wish to add a collaborator</p>
+    <p><strong>NOTE: </strong>Collaborator must already be registered with Blocipedia</p>
+  </div>
+  <div class="col-md-7">
+    <%= form_for [@wiki, @collaborator] do |f| %>
+      <div class="form-group">
+        <%= f.label :email %>
+        <%= f.email_field :user_id, autofocus: true, class: 'form-control', placeholder: "Enter email" %>
```

Comment:
Here is the other part where I had major issues.  For this form, I wanted to be able for the user to input an email address, then have the controller find the proper user id to add as a collaborator via the email address.  Was able to get this to work but only by stating that the email field above was passing the information to the `user_id` param.  If I instead tried `f.email_field :email` I would get a `no method` error (presumably because `email` is not an appropriate attribute for the collaborator model).  Is there another way I should have created this form so the code doesn't read like I'm asking for a user id?